README.md

```diff
@@ -269,7 +271,7 @@ The Service Binding resource **MAY** define `.spec.application.containers`, as a
 
 A Service Binding Resource **MAY** define a `.spec.env` which is an array of `EnvMapping`.  An `EnvMapping` object **MUST** define `name` and `key` entries.  The `key` of an `EnvMapping` **MUST** refer to a binding `Secret` key name.  The value of this `Secret` entry **MUST** be configured as an environment variable on the resource represented by `application`.
 
-A Service Binding resource **MUST** define `.status.conditions` which is an array of `Condition` objects as defined in [meta/v1 Condition][mv1c].  At least one condition containing a `type` of `Ready` **MUST** be defined.  The `Ready` condition **SHOULD** contain appropriate values defined by the implementation.  As label selectors are inherently queries that return zero-to-many resources, it is **RECOMMENDED** that `ServiceBinding` authors use a combination of labels that yield a single resource, but implementors **MUST** handle each matching resource as if it was specified by name in a distinct `ServiceBinding` resource. Partial failures **MUST** be aggregated and reported on the binding status's `Ready` condition. A Service Binding resource **SHOULD** reflect the secret projected into the application as `.status.binding.name`.
```

Comment:
This recommendation is applicable to applications' labels selectors, right? And this PR is about label selectors for services. Now there are two kinds of label selectors, we should rephrase this sentence for clarity emphasizing that this part is concerning application label selectors.

Reply:
The previous section was about service binding in general, but it called out multiple applications because that was the old behaviour.  We're now saying only 1 match should be taken, so the deleted section no longer applies to either application or service.